### PR TITLE
Correct some typedef documentation

### PR DESCRIPTION
**Pull request by levitte**

### CODE REVIEW

**Review by mattcaswell — APPROVED** [review]

File: doc/man3/BIO_ctrl.pod [hunk 1]
```diff
@@ -12,8 +12,8 @@ BIO_get_info_callback, BIO_set_info_callback, bio_info_cb
 
  #include <openssl/bio.h>
 
- typedef void (*bio_info_cb)(BIO *b, int oper, const char *ptr,
-                             int arg1, long arg2, long arg3);
+ typedef void bio_info_cb(BIO *b, int oper, const char *ptr,
+                          int arg1, long arg2, long arg3);
 
  long BIO_ctrl(BIO *bp, int cmd, long larg, void *parg);
  long BIO_callback_ctrl(BIO *b, int cmd, bio_info_cb cb);
```
Comment (bernd-edlinger) [inline doc/man3/BIO_ctrl.pod]: this should use `bio_info_cb *cb` as a consequnece.

Comment (levitte) [reply doc/man3/BIO_ctrl.pod]: Thanks.  Fixed, and I looked through the other corrected typedefs as well.

**Comment by levitte** [comment]
Correction needs reconfirm

**Review by mattcaswell — APPROVED** [review]
Reconfirm

**Review by bernd-edlinger — APPROVED** [review]

**Comment by bernd-edlinger** [comment]
doc-nits?
doc/man3/EVP_PKEY_keygen.pod:1: *ctx) missing from NAME section
doc/man3/EVP_PKEY_keygen.pod:1: EVP_PKEY_gen_cb missing from SYNOPSIS
doc/man3/PEM_read_bio_PrivateKey.pod:1: *u) missing from NAME section
doc/man3/PEM_read_bio_PrivateKey.pod:1: pem_password_cb missing from SYNOPSIS
make: *** [doc-nits] Error 1

**Comment by levitte** [comment]
Ah, right, `util/find-doc-nits` doesn't understand function signature typedefs.  Fix coming up

**Comment by levitte** [comment]
find-doc-nits corrected

**Review by bernd-edlinger — APPROVED** [review]
re-confirmed, thanks!

**Comment by levitte** [comment]
Ok, doc-nits passes too...  are we ready for a final reconfirm?

**Review by mattcaswell — APPROVED** [review]
Reconfirm!

**Comment by levitte** [comment]
Merged.

master:  5bf6d418034a246bd3680d648c22e2c4500a3e0a and 0ed78e78007bb74e48e6f59fa2388bb244153bf0
1.1.0: 5bf6d418034a246bd3680d648c22e2c4500a3e0a and 0ed78e78007bb74e48e6f59fa2388bb244153bf0

(for 1.1.0, some please explain why `util/find-doc-nits` was removed in 74c29dc6eac1f6f8a8c2ca28f8c01a43beb8c15b .  @richsalz?)

**Comment by levitte** [comment]
Merged.

master:  5bf6d418034a246bd3680d648c22e2c4500a3e0a and 0ed78e78007bb74e48e6f59fa2388bb244153bf0
1.1.0: 5c3f01ff1a94bdb0d32bbb08f69f34316eaa136c and 125993dee9f7e7361dd7dcf75ddb34050a78d973

(for 1.1.0, some please explain why `util/find-doc-nits` was removed in 74c29dc6eac1f6f8a8c2ca28f8c01a43beb8c15b .  @richsalz?)

**Comment by richsalz** [comment]
because in 1.1.0 it's find-doc-nits.pl ?

**Comment by levitte** [comment]
That one was removed in 8d92311c50e58dee074bbbf7fa390e592ee8df77

In both cases, it looks like they got removed as part of a cherry-pick (i.e. cherry-pick gone rogue).

**Comment by levitte** [comment]
D'oh...  sorry, it got renamed in 8d92311c50e58dee074bbbf7fa390e592ee8df77

I need to read the *whole* commit before speaking...

**Comment by levitte** [comment]
74c29dc6eac1f6f8a8c2ca28f8c01a43beb8c15b still looks like a chery-pick gone rogue, though...

**Pull request closed without merging**